base_import_async/models/base_import_async.py

```diff
@@ -30,7 +30,9 @@
 from openerp.models import fix_import_export_id_paths
 from openerp.tools.translate import _
 
-from openerp.addons.connector.queue.job import job, related_action
+from openerp.addons.connector.queue.job import (
```

Comment:
Yeah, but with a simple try we also prevent an Odoo break and everyone happy :smile:
It's similar as protecting external dependencies.